packages/core/src/renderer/components/cluster-settings/__tests__/icon-settings.test.tsx

```diff
@@ -39,6 +40,8 @@ const clusterEntity = new KubernetesCluster({
   },
 });
 
+cluster.preferences.clusterName = "some-cluster-name";
```

Comment:
Looking at `ClusterModel` (first argument of `Cluster`) it seems you can pass `preferences` in the constructor above.

```ts
export interface ClusterModel {
  /** Unique id for a cluster */
  id: ClusterId;

  /** Path to cluster kubeconfig */
  kubeConfigPath: string;

  /** User context in kubeconfig  */
  contextName: string;

  /** Preferences */
  preferences?: ClusterPreferences;

  /** Metadata */
  metadata?: ClusterMetadata;

  /** List of accessible namespaces */
  accessibleNamespaces?: string[];

  /**
   * Labels for the catalog entity
   */
  labels?: Partial<Record<string, string>>;
}
```